docs: Make minor changes to the Export to Excel articles (#2800)
DevExpress/devextreme-reactive: packages/dx-react-grid/docs/guides/export.md

4. Save the file

>    To save the file to the user's local storage, implement the `onSave` callback of the `GridExporter` component. In the following code, the file is saved using the `file-saver` package, but you can use any other package that provides similar functionality.

Reviewer: Can we instead say:
To save the file to local storage at runtime, use the `onSave` callback for the `GridExporter` component. In the following code, the file is saved via the `file-saver` package, but you can use any other similar package.

Author: It's important that the local storage is the user's, not the developer's. In addition, the phrase "use the callback for the component" doesn't sound right. "**Implement** the callback for the component" does, so I suggest the following version:

```suggestion
   To save the file to the user's local storage at runtime, implement the `onSave` callback for the `GridExporter` component. In the following code, the file is saved via the `file-saver` package, but you can use any other similar package.
```

Reviewer: I see. Then let's change it to:
To save a file to a user's local storage at runtime, implement the `onSave` callback for the `GridExporter` component.

## Supported Grid Features

> Data shaping settings applied to the Grid can be also applied to the exported Excel document. For this, pass Grid settings to the following `GridExporter` properties:

Reviewer: Can we say:
To apply a grid's data shaping settings to an exported Excel document, pass the settings to the following `GridExporter` properties:

Author: ```suggestion
To apply the Grid's data shaping settings to an exported Excel document, pass the settings to the following `GridExporter` properties:
```
